<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NHSCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CalcSumBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SUM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowHexResultBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hex Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowDecimalResultBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal Value</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="HexInputBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="InputTextLengthLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CalcXorBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
